core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -359,4 +360,71 @@ private TableProperties() {}
 
   public static final String UPSERT_ENABLED = "write.upsert.enabled";
   public static final boolean UPSERT_ENABLED_DEFAULT = false;
+
+  /** @deprecated will be removed in 1.3.0, use the HMS_TABLE_OWNER constant from HiveCatalog */
+  @Deprecated public static final String HMS_TABLE_OWNER = "hive.metastore.table.owner";
```

Comment:
why is this being added? probably a leftover from a rebase

Reply:
Yes, this was leftover. Corrected it.